linera-base/src/identifiers.rs

```diff
@@ -38,8 +38,7 @@ pub enum AccountOwner {
     /// 32-byte account address.
     Address32(CryptoHash),
     /// 20-byte account EVM-compatible address.
-    #[debug(with = "hex_debug")]
-    Address20([u8; 20]),
+    Address20(#[debug(with = "hex_debug")] [u8; 20]),
```

Comment:
I would add a test for `Address20` as well as `Address32` (which I assume is correct right now, just unsure because of a lack of test).

Reply:
Good idea! Turns out:
* For `Address32` we only print 8 bytes because of the `Debug` impl for `CryptoHash`. I guess I should change that, too, and print the whole hash.
* For `Reserved`, we print it in decimal. I guess that's okay, although I'd prefer hexadecimal there, too.

Reply:
For `Reserved`, we currently have only 0 as a possibility.
This corresponds to `CHAIN`, so I guess that we can do something here as well.

Reply:
Done in https://github.com/linera-io/linera-protocol/pull/4453/commits/e0abfd04f29e6986313b9add6f246434721453c1.